cmd/template-resolver/utils/templateResolver.go

```diff
@@ -141,7 +140,7 @@ func ProcessTemplate(yamlBytes []byte, hubKubeConfigPath, clusterName string) ([
 			LookupNamespace: policy.GetNamespace(),
 		}
 
-		hubResolver, err = templates.NewResolver(hubKubeConfig, hubTemplatesConfig)
+		hubResolver, err = templates.NewResolver(hubKubeConfig, defaultHubTemplateOpts.config)
```

Comment:
Musing/optional: I wonder if we should just go ahead and egregiously blanket resolve all hub templates here for simplicity.